lib/foreman_maintain/reporter/cli_reporter.rb

```diff
@@ -117,7 +117,11 @@ def ask(message, options = {})
         # add space at the end as otherwise highline would add new line there :/
         message = "#{message} " unless message =~ /\s\Z/
         answer = @hl.ask(message) { |q| q.echo = false if options[:password] }
-        answer.to_s.chomp.downcase if answer
+        if answer
+          answer.to_s.chomp!
+          answer.downcase! unless options[:username] or options[:password]
```

Comment:
`or` & `and`  are used as `control flow operators`, not as boolean operators.
So, use `||` operator here.
Please modify syntax like  `options[:username] || options[:password]` 
instead of `options[:username] or options[:password]`